database-plugins/pom.xml

```diff
@@ -88,6 +92,11 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-jdbc</artifactId>
+      <version>${opentracingjdbc.version}</version>
+    </dependency>
```

Comment:
nit: Put this dependency before line no 49 as the comment says test dependencies start after it.